_layouts/sip.html

```diff
@@ -1,12 +1,15 @@
 ---
-layout: inner-page-parent-dropdown
+layout: root-content-layout
 includeTOC: true
 ---
 
 <section class="content">
 	<div class="wrap">
 		<div class="content-primary documentation">
 			<div class="inner-box toc-context">
+				{% if page.new-version %}
```

Comment:
I am not sure we need this in the `sip` layout

Reply:
done